CMakeLists.txt

```diff
@@ -69,6 +70,9 @@ if(NOT CMAKE_BUILD_TYPE)
       FORCE)
 endif()
 
+if(WITH_AMD_GPU)
+endif()
```

Comment:
line 73-74 could be removed since there is nothing between them.

Reply:
Removed.